### DISCUSSION TITLE
Pyspark connector

### DISCUSSION THREAD

**Discussion by lucifermorningstar1305** [kuzudb/kuzu#4369] [Ideas]
While dealing with large scale data it's best if there is a way to integrate kuzu with spark dataframe. Something like what Neo4j has. This way anyone can upload batches of data to Kuzu without writing extensive code.

**Comment by prrao87** [comment]
Hi @lucifermorningstar1305, this is already an existing feature request, so we'd much appreciate if you could add this as a comment on the ongoing issue here: https://github.com/kuzudb/kuzu/issues/3774

Please also upvote the issue (+1) so that we can use this information for our prioritization purposes.
